docs/en/operations/utilities/clickhouse-local.md

```diff
@@ -34,21 +34,27 @@ The binary you just downloaded can run all sorts of ClickHouse tools and utiliti
 
 A common use of `clickhouse-local` is to run ad-hoc queries on files: where you don't have to insert the data into a table. `clickhouse-local` can stream the data from a file into a temporary table and execute your SQL.
 
-If the file is sitting on the same machine as `clickhouse-local`, use the `file` table engine. The following `reviews.tsv` file contains a sampling of Amazon product reviews:
+If the file is sitting on the same machine as `clickhouse-local`, you can simple specify the file to load. The following `reviews.tsv` file contains a sampling of Amazon product reviews:
 
 ```bash
-./clickhouse local -q "SELECT * FROM file('reviews.tsv')"
+./clickhouse-local -q "SELECT * FROM 'reviews.tsv'"
```

Comment:
JFYI, the reason why instead of clickhouse-local it was written as `./clickhouse local` is as follows:

Our documentation engineers (all two of them) are using Mac.

In Mac, the ClickHouse installation script `Install.cpp` does not support creating symlinks and installing into /usr/bin/

But ClickHouse download script will put the `clickhouse` binary into the current directory, and it can be run without installation.

Although, I figured out, how to install ClickHouse on Mac here: https://github.com/ClickHouse/NoiSQL#quick-start---macos 
we did not put it into the Install script because no one among the engineers is using Mac, so the development iterations for Mac installation can be long.

Your edit does not look right because it is unlikely that the symlink exists, but clickhouse is not installed in PATH.